OpenImis.ePayment/Escape/Payment/Data/GepgUtility.cs

```diff
@@ -632,57 +514,39 @@ public async Task<string> CreateBulkBills(IConfiguration configuration, RequestB
 
                 billTrxInf.PyrId = dr["BillId"].ToString();
                 billTrxInf.PyrName = "CHF IMIS";
-                billTrxInf.PyrEmail = "info@imis.co.tz";  
+                billTrxInf.PyrEmail = "info@imis.co.tz";
                 billTrxInf.PyrCellNum = "";
 
                 billTrxRefs.Add(billTrxInf);
 
             }
 
-
             string accountCode = GetAccountCodeByProductCode(model.ProductCode);
 
             newBill = new gepgBillSubReq();
             newBill.BillHdr = new BillHdr() { SpCode = accountCode, RtrRespFlg = true };
             newBill.BillTrxInf = billTrxRefs;
 
+            var xs = new XmlSerializer(typeof(gepgBillSubReq));
+            var ns = new XmlSerializerNamespaces();
+            ns.Add("", "");
+            var settings = new XmlWriterSettings() { OmitXmlDeclaration = true };
 
-            XmlSerializer xs = null;
-            XmlSerializerNamespaces ns = null;
-            XmlWriterSettings settings = null;
-            String outString = String.Empty;
-
-            XmlWriter xw = null;
-
-            try
+            using (var sb = new MemoryStream())
+            using (var xw = XmlWriter.Create(sb, settings))
             {
-                settings = new XmlWriterSettings();
-                settings.OmitXmlDeclaration = true;
-                //settings.Indent = true;
-                ns = new XmlSerializerNamespaces();
-                ns.Add("", "");
-
-                StringBuilder sb = new StringBuilder();
-                xs = new XmlSerializer(typeof(gepgBillSubReq));
-
-                xw = XmlWriter.Create(sb, settings);
-
-                xs.Serialize(xw, newBill, ns);
-                xw.Flush();
-                outString = sb.ToString();
-            }
-            catch (Exception ex)
-            {
-                Console.WriteLine(ex.ToString());
-            }
-            finally
-            {
-                if (xw != null)
+                try
                 {
-                    xw.Close();
+                    xs.Serialize(xw, newBill, ns);
+                    xw.Flush();
                 }
+                catch (Exception ex)
+                {
+                    Console.WriteLine(ex.ToString());
```

Comment:
Instead of logging the error in Console, can we log it to Log4Net or somewhere else to review the exception later?

Reply:
The same as above, i didn't want to modify the component to allow logging